.github/workflows/frontend_builder.yml

```diff
@@ -0,0 +1,42 @@
+name: Frontend Builder
+
+on:
+  workflow_dispatch:
+  
+  push:
+    branches: [actions]
```

Comment:
Currently, set this to `actions` branch to test, but if this looks good to you I can change it to `production`.